types/web3/eth/index.d.ts

```diff
@@ -166,12 +166,12 @@ export default interface Eth {
     ): Promise<EncodedTransaction>;
     sendSignedTransaction(
         data: string,
-        cb?: Callback<string>
-    ): PromiEvent<TransactionReceipt>;
+        cb?: Callback<TransactionReceipt | string | { confNumber: number, receipt: TransactionReceipt }>
```

Comment:
I am sorry, but for the callback I do not see multiple possible values. As far as I understand the callback will only be called with a `string` as the `result`. The transaction hash...

Reply:
@Levino Agreed.  So I just added some tests.